zetaclient/tss/tss_signer.go

```diff
@@ -419,17 +413,24 @@ func (tss *TSS) SignBatch(
 	return sigBytes, nil
 }
 
-// Validate validates the TSS
-func (tss *TSS) Validate() error {
+// ValidateAddresses validates the TSS EVM and BTC addresses
+func (tss *TSS) ValidateAddresses(btcChainIDs []int64) error {
```

Comment:
Can you add an explanation for this method? I don't have much context and it's hard to grasp what's going on here 😅

Reply:
Gotta. I'm going to add a bit details.

Reply:
https://github.com/zeta-chain/node/pull/2907/commits/7c5fbb943924c4cc69fda434a8599661bbaa4aff